microsoft-edge/devtools-guide-chromium/json-viewer/json-viewer.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
Reformat the links in
https://github.com/MicrosoftEdge/Demos/tree/main/json-dummy-data#readme
as githit.io links, like done in the body of this article; 
https://microsoftedge.github.io/Demos/json-dummy-data/256KB-min.json
I think I'm seeing those json files formatted due to GH, not due to Edge.

The above line is probably a sufficient template, but here's the xform'n pattern:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/sample-code/sample-code#url-patterns-for-rendered-demo-webpages-and-source-code